src/data/dailylimits.txt

```diff
@@ -8,6 +8,7 @@ Cast	Advanced Saucecrafting	reagentSummons
 Cast	Ancestral Recall	_ancestralRecallCasts	10
 Cast	Army of Toddlers	_armyToddlerCast
 Cast	Asdon Martin: Missile Launcher	_missileLauncherUsed
+Cast    Assert your Authority   _authorityUsed  3
```

Comment:
should be tab separated

Reply:
Odd; I am tab separating them. Is this a VSCode thing?

Reply:
if you are tab separating them, VSCode or something else is converting your tabs to spaces.

From what I recall, it is one of the options in the bottom right.